metis-validation/metis-validation-rest/src/main/resources/schema/RDF.xsd

```diff
@@ -183,4 +183,73 @@
 			</extension>
 		</simpleContent>
 	</complexType>
-</schema>
\ No newline at end of file
+
+	<complexType name="RightsStatements">
+		<attribute name="resource" use="required" form="qualified">
+			<simpleType>
+				<restriction base="string">
+					<!-- 
+					CC VERSION 1.0 LICENSES
+					<http://creativecommons.org/licenses/[PERMISSIONS]/1.0/[PORT]/>
+					PERMISSIONS: by | by-sa | by-nd | by-nc | by-nc-sa | by-nc-nd | by-nd-nc 
+					PORT: generic (no port) | fi | il | nl
+					-->
+					<pattern value="http[:]//creativecommons[.]org/licenses/(by|by-sa|by-nd|by-nc|by-nc-sa|by-nc-nd|by-nd-nc)/1[.]0/(fi|il|nl)?"/>
+
+					<!-- 
+					CC VERSION 2.0 LICENSES
+					<http://creativecommons.org/licenses/[PERMISSIONS]/2.0/[PORT]/>
+					PERMISSIONS: by | by-sa | by-nd | by-nc | by-nc-sa | by-nc-nd
+					PORT: generic (no port) | au | at | be | br | ca | cl | hr | uk | fr | de | it | jp | nl | pl | kr | es | tw
+					-->
+					<pattern value="http[:]//creativecommons[.]org/licenses/(by|by-sa|by-nd|by-nc|by-nc-sa|by-nc-nd)/2[.]0/(au|at|be|br|ca|cl|hr|uk|fr|de|it|jp|nl|pl|kr|es|tw)?"/>
+
+					<!-- 
+					CC VERSION 2.1 LICENSES
+					<http://creativecommons.org/licenses/[PERMISSIONS]/2.1/[PORT]/>
+					PERMISSIONS: by | by-sa | by-nd | by-nc | by-nc-sa | by-nc-nd
+					PORT: au | es | jp
+					-->
+					<pattern value="http[:]//creativecommons[.]org/licenses/(by|by-sa|by-nd|by-nc|by-nc-sa|by-nc-nd)/2[.]1/(au|es|jp)"/>
```

Comment:
Missing the ? at the end of the pattern here?

Reply:
no check document.

Reply:
ok